### ISSUE TITLE
Recyclable Waste Identification Using CNN Image Recognition and Gaussian Clustering. (arXiv:2011.01353v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#331]
Recyclable Waste Identification Using CNN Image Recognition and Gaussian Clustering. (arXiv:2011.01353v1 [cs.CV])<br>
https://ift.tt/3mNY9x5<br>
<p>Waste recycling is an important way of saving energy and materials in the production process. In general cases recyclable objects are mixed with unrecyclable objects, which raises a need for identification and classification. This paper proposes a convolutional neural network (CNN) model to complete both tasks. The model uses transfer learning from a pretrained Resnet-50 CNN to complete feature extraction. A subsequent fully connected layer for classification was trained on the augmented TrashNet dataset [1]. In the application, sliding-window is used for image segmentation in the pre-classification stage. In the post-classification stage, the labelled sample points are integrated with Gaussian Clustering to locate the object. The resulting model has achieved an overall detection rate of 48.4% in simulation and final classification accuracy of 92.4%.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
